bitnami/geode/values.yaml

```diff
@@ -0,0 +1,1027 @@
+  ## @param server.log4j Specify content for log4j2.xml on Cache server nodes (optional)
+  ##
+  log4j: {}
+  ## @param server.existingLog4jConfigMap Name of existing ConfigMap containing a custom log4j2.xml configuration for Cache server
+  ## NOTE: When it's set the server.log4j is ignored
+  ##
+  existingLog4jConfigMap: ""
+  ## @param server.replicaCount Number of Cache Server replicas to deploy
+  ##
+  replicaCount: 3
+  ## @param server.containerPorts.server Cache Server container port
+  ## @param server.containerPorts.http Cache Server HTTP container port
+  ## @param server.containerPorts.rmi Cache Server RMI container port
+  ## @param server.containerPorts.metrics Cache Server internal metrics container port
+  ##
+  containerPorts:
+    server: 40404
+    http: 7070
+    rmi: 1099
+    metrics: 9915
+  ## Configure extra options for Cache server containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param server.livenessProbe.enabled Enable livenessProbe on Cache server containers
+  ## @param server.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param server.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param server.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param server.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param server.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: false
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 3
+    failureThreshold: 3
+    successThreshold: 1
+  ## @param server.readinessProbe.enabled Enable readinessProbe on Cache server containers
+  ## @param server.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param server.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param server.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param server.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param server.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: false
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 4
+    failureThreshold: 3
+    successThreshold: 1
+  ## @param server.startupProbe.enabled Enable startupProbe on Cache server containers
+  ## @param server.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param server.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param server.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param server.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param server.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: true
```

Comment:
We want the startup disabled by default because of compatibilities with k8s versions and Openshift